docs/production/setting-up-javascript.md

```diff
@@ -1,59 +1,115 @@
+**Some components cannot be initialised by this method** and this is noted on their individual documentation pages. For more details, see how to [initialise individual components](#initialise-individual-components) below.
+
+### Serve the JavaScript files from the combined javascripts folders – recommended
+
+Set up your routing so requests for the JavaScript files in `<YOUR-SITE-URL>/javascripts` are served from both `/node_modules/govuk-frontend/dist/govuk` and `/node_modules/@ministryofjustice/frontend/moj`.
+
+You will also need to install and serve [jQuery](https://jquery.com/).
+
+If you’re using [express.js](https://expressjs.com/), request routing could be set up as follows:
+
+```js
+router.use('/javascripts', [
+  express.static('node_modules/@ministryofjustice/frontend/moj'),
+  express.static('node_modules/govuk-frontend/dist/govuk'),
+  express.static('node_modules/jquery/dist'),
+])
+```
+
+### Copy the JavaScript files into your application
+
+If you decide to copy the JavaScript files instead, copy `/node_modules/@ministryofjustice/frontend/moj/moj-frontend.min.js`, `/node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.js` and `/node_modules/jquery/dist/jquery.min.js` files into your application.
+
+You should use an automated task or your build pipeline to copy the files, so your project folder stays up to date when updates to MoJ Frontend and GOV.UK Frontend are released.
+
+## Import JavaScript using a bundler
+
+If you decide to import using a bundler like [Rollup](https://rollupjs.org/) or [webpack](https://webpack.js.org/), the bundled JavaScript files must be added using `<script type="module">` in your HTML:
+
+```html
+<script type="module" src="/javascripts/application.min.js"></script>
+```
 
-## Import using a bundler
+We encourage the use of ECMAScript (ES) modules, but you should check your bundler does not unnecessarily downgrade modern JavaScript for unsupported browsers.
 
-If you decide to import using a bundler, use `import` to import jQuery and all of the design systems’s components, then run the `initAll` function to initialise them:
+If you decide to import using a bundler, use `import` to import jQuery and all of the design systems’s components, then run the `initAll` functions to initialise them:
 
 ```mjs
-import { initAll } from '@ministryofjustice/frontend'
+import * as MOJFrontend from '@ministryofjustice/frontend'
+import * as GOVUKFrontend from 'govuk-frontend'
 import $ from 'jquery'
 
 window.$ = $
-initAll()
+GOVUKFrontend.initAll()
+MOJFrontend.initAll()
 ```
 
-## Initialize individual components
+### Initialise individual components
 
 Rather than using `initAll`, you can initialise individual components by identifying them with their `data-module` attribute. For example, to initialise just the [Password reveal](/components/password-reveal/) component:
```

Comment:
```suggestion
Rather than using `initAll`, you can initialise individual components by identifying them with their `data-module` attribute. For example, to initialise just the [Date picker](/components/date-picker/) component:

```html
<script>
  var DatePicker = window.MOJFrontend.DatePicker
  var $datePicker = document.querySelector(
    '[data-module="moj-date-picker"]'
  )
  if ($datePicker) {
    new DatePicker($datePicker)
  }
</script>
```
```